src/index.css

```diff
@@ -1,14 +1,383 @@
+    url('/public/fonts/FuturaPT/FuturaPT-ExtraBold.woff') format('woff'),
+    url('/public/fonts/FuturaPT/FuturaPT-ExtraBold.ttf') format('truetype');
+  font-weight: 800;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Futura PT Cond';
+  src: url('/public/fonts/FuturaPT/FuturaPTCond-BoldObl.eot');
+  src: local('Futura PT Cond Bold Oblique'), local('FuturaPTCond-BoldObl'),
+    url('/public/fonts/FuturaPT/FuturaPTCond-BoldObl.eot?#iefix') format('embedded-opentype'),
+    url('/public/fonts/FuturaPT/FuturaPTCond-BoldObl.woff') format('woff'),
+    url('/public/fonts/FuturaPT/FuturaPTCond-BoldObl.ttf') format('truetype');
+  font-weight: bold;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Futura PT Cond';
+  src: url('/public/fonts/FuturaPT/FuturaPTCond-MediumObl.eot');
+  src: local('Futura PT Cond Medium Oblique'), local('FuturaPTCond-MediumObl'),
+    url('/public/fonts/FuturaPT/FuturaPTCond-MediumObl.eot?#iefix') format('embedded-opentype'),
+    url('/public/fonts/FuturaPT/FuturaPTCond-MediumObl.woff') format('woff'),
+    url('/public/fonts/FuturaPT/FuturaPTCond-MediumObl.ttf') format('truetype');
+  font-weight: 500;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Futura PT Cond Book';
+  src: url('/public/fonts/FuturaPT/FuturaPTCond-BookObl.eot');
+  src: local('Futura PT Cond Book Oblique'), local('FuturaPTCond-BookObl'),
+    url('/public/fonts/FuturaPT/FuturaPTCond-BookObl.eot?#iefix') format('embedded-opentype'),
+    url('/public/fonts/FuturaPT/FuturaPTCond-BookObl.woff') format('woff'),
+    url('/public/fonts/FuturaPT/FuturaPTCond-BookObl.ttf') format('truetype');
+  font-weight: normal;
+  font-style: italic;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
 body {
+  line-height: 1;
   margin: 0;
   padding: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: "Futura PT", Helvetica, Arial, sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-size: 16px;
```

Comment:
For `rem` to work, the base font size must be set on the `html` element, not `body` element.

```css
html {
    font-size: 16px;
}
```